async-query-core/src/main/java/org/opensearch/sql/spark/dispatcher/model/JobType.java

```diff
@@ -8,6 +8,7 @@
 public enum JobType {
   INTERACTIVE("interactive"),
```

Comment:
Should we revisit the definitions of these jobtypes? What is the difference between a streaming and refresh job type.